src/entity/Guild/GuildUser.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Entity,
+  Column,
+  OneToOne,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn
+} from 'typeorm'
+import { Guild } from '.'
+import { GuildUserWarning } from './GuildUserWarning'
+import { GuildUserKick } from './GuildUserKick'
+import { GuildUserBan } from './GuildUserBan'
+import { GuildUserReputation } from './GuildUserReputation'
```

Comment:
Add the classes from lines 10-13 to the index.ts file so the imports can be combined on line 9.